plugins/dashboard/routes.go

```diff
@@ -101,7 +101,7 @@ func passThroughAPIRoute(e echo.Context) error {
 func passThroughSpammerRoute(e echo.Context) error {
 	apiBindAddr := deps.NodeConfig.String(restapi.CfgRestAPIBindAddress)
 
-	data, err := readDataFromURL("http://" + apiBindAddr + "/spammer?" + e.Request().URL.RawQuery)
+	data, err := readDataFromURL("http://" + apiBindAddr + "/api/plugins/spammer?" + e.Request().URL.RawQuery)
```

Comment:
This probably can be removed if the dashboard adapts to using the api/plugins route

Reply:
Why was it there in the first place? Maybe I will leave it that way and you will change it anyway with the JWT stuff, right?